File: python/FindCenterofStarGraph.py
from typing import List

def findCenter(edges: List[List[int]]) -> int:
    first_edge, second_edge = edges[0], edges[1]
    return first_edge[0] if first_edge[0] in second_edge else first_edge[1]

# Example usage:
edges =  [[1,2],[2,3],[4,2]]
center = findCenter(edges)
print(center)  # Output should be 2

File: JAVA/java/src/FindCenterofStarGraph.java
import java.util.List;

public class FindCenterofStarGraph {
    public static int findCenter(List<List<Integer>> edges) {
        int[] firstEdge = new int[] { edges.get(0).get(0), edges.get(0).get(1) };
        int[] secondEdge = new int[] { edges.get(1).get(0), edges.get(1).get(1) };

        return firstEdge[0] == secondEdge[0] || firstEdge[0] == secondEdge[1] ? firstEdge[0] : firstEdge[1];
    }

    public static void main(String[] args) {
        List<List<Integer>> edges = List.of(
                List.of(1, 2),
                List.of(2, 3),
                List.of(4, 2)
        );

        int center = findCenter(edges);
        System.out.println(center);  // Output should be 2
    }
}

